.bazelci/presubmit.yml

```diff
@@ -17,4 +17,130 @@ platforms:
     - "--"
     - "..."
     - "-//tests/docker/..."
-
+  rbe_ubuntu1604:
+    build_targets:
+    - "--"
```

Comment:
add a comment that we are selecting to test only the targets that build on RBE because the default image does not support docker

Reply:
Done.